src/lib/routing/lib/slug.test.ts

```diff
@@ -0,0 +1,56 @@
+import { describe, expect, test } from 'vitest';
+import type { PageRouteFragment, SiteLocale } from '@lib/datocms/types';
+import { getLocalizedSlug, getSlugFromPath } from './slug';
+
+const mockLocales = ['en', 'nl'] as SiteLocale[];
+const pageSlugs = {
+  en: 'example-page',
+  nl: 'voorbeeld-pagina'
+} as Record<string, string> as Record<SiteLocale, string>;
```

Comment:
Make sure that this test scenario works, even when the SiteLocale definition generated from a DatoCMS instance might have completely different locales